runtime/service_integ_test.go

```diff
@@ -305,10 +305,11 @@ func testMultipleVMs(ctx context.Context, t *testing.T, count int) {
 			MaxContainers: 3,
 
 			JailerConfig: &proto.JailerConfig{
-				UID:        300000,
-				GID:        300000,
-				NetNS:      netns.Path(),
-				CgroupPath: "/mycgroup",
+				UID:           300000,
+				GID:           300000,
+				NetNS:         netns.Path(),
+				CgroupPath:    "/mycgroup",
+				CgroupVersion: "2",
```

Comment:
Does it really work? The newly-added CgroupVersion field won't be used automatically.

Reply:
yes, it helped me to resolve the test **TestMultipleVMs_Isolated** and I have added **CgroupVersion** field using `firecracker.proto` file then `make proto` helped in auto-generating the field in `firecracker.pb.go` file.

Reply:
Pipeline works fine after removing Cgroupversion.